samples/javascript_nodejs/21.corebot-app-insights/bots/resources/welcomeCard.json

```
@@ -1,46 +1,46 @@
 {
-    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
-    "type": "AdaptiveCard",
-    "version": "1.0",
-    "body": [
-      {
-        "type": "Image",
-        "url": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQtB3AwMUeNoq4gUBGe6Ocj8kyh3bXa9ZbV7u1fVKQoyKFHdkqU",
-        "size": "stretch"
-      },
-      {
-        "type": "TextBlock",
-        "spacing": "medium",
-        "size": "default",
-        "weight": "bolder",
-        "text": "Welcome to Bot Framework!",
-        "wrap": true,
-        "maxLines": 0
-      },
-      {
-        "type": "TextBlock",
-        "size": "default",
-        "isSubtle": "true",
-        "text": "Now that you have successfully run your bot, follow the links in this Adaptive Card to expand your knowledge of Bot Framework.",
-        "wrap": true,
-        "maxLines": 0
-      }
-    ],
-    "actions": [
-      {
-        "type": "Action.OpenUrl",
-        "title": "Get an overview",
-        "url": "https://docs.microsoft.com/en-us/azure/bot-service/?view=azure-bot-service-4.0"
-      },
-      {
-        "type": "Action.OpenUrl",
-        "title": "Ask a question",
-        "url": "https://stackoverflow.com/questions/tagged/botframework"
-      },
-      {
-        "type": "Action.OpenUrl",
-        "title": "Learn how to deploy",
-        "url": "https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-deploy-azure?view=azure-bot-service-4.0"
-      }
-    ]
-  }
\ No newline at end of file
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.0",
+  "body": [
+    {
+      "type": "Image",
+      "url": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQtB3AwMUeNoq4gUBGe6Ocj8kyh3bXa9ZbV7u1fVKQoyKFHdkqU",
+      "size": "stretch"
+    },
+    {
+      "type": "TextBlock",
+      "spacing": "medium",
+      "size": "default",
+      "weight": "bolder",
+      "text": "Welcome to Bot Framework!",
+      "wrap": true,
+      "maxLines": 0
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "isSubtle": true,
+      "text": "Now that you have successfully run your bot, follow the links in this Adaptive Card to expand your knowledge of Bot Framework.",
+      "wrap": true,
+      "maxLines": 0
+    }
+  ],
+  "actions": [
+    {
+      "type": "Action.OpenUrl",
+      "title": "Get an overview",
+      "url": "https://docs.microsoft.com/en-us/azure/bot-service/?view=azure-bot-service-4.0"
```

Need to remove the local from the URL links.

```suggestion
      "url": "https://docs.microsoft.com/azure/bot-service/?view=azure-bot-service-4.0"
```